cicd/build.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash -e
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
 # Copyright 2021 VMware, Inc.
```

Comment:
You've probably noticed the bot is prepending the new notice directly-
am now cleaning up the double and sometimes even triple notices (leftovers from prev notice format update).

Reply:
Changing one digit is easy, I agree, it's just the part we have to remember to do it every year that bugs me. Anyway, I think you've discovered a much worse problem that deserves a ticket.

Reply:
another leftover variation would be:
`# Copyright 2021 VMware, Inc.
# SPDX-License-Identifier: Apache-2.0
# Copyright 2022 VMware, Inc.
# SPDX-License-Identifier: Apache-2.0
`

Reply:
I also noticed the .yaml & .yml files are not configured for handling,
yet they (at least some) had old notices added. I will be addressing that in the pre-commit hook reconfigured PR https://github.com/vmware/versatile-data-kit/pull/1625